x-pack/plugins/viz_editor/server/query/where_operations.ts

```diff
@@ -129,15 +129,15 @@ function parseComparison([a, b, ...etc]: any[]) {
   const { operation: op1, argument: arg1 } = a;
   const { operation: op2, argument: arg2 } = b;
 
-  if (op1 !== 'col' && op2 !== 'col') {
+  if (op1 !== 'column' && op2 !== 'column') {
```

Comment:
@chrisdavies This test wasn't throwing typescript errors, which is potentially a problem with strings-as-types. The only alternative I know if is to define structural types such as `interface Operation<Type> { [Type]: true }`, what do you think?

Reply:
While I haven't fully solved this in the latest version, it should be stricter now than it was before.

Reply:
I think it's OK to not be 100% covered by the type system. 100% coverage is a noble goal, but I never had it in C# or C++ projects, and I doubt we'll get it in TypeScript. (PureScript or Reason, maybe? :) If we can come up with elegant ways to get type coverage, that's a win, but it's not something I'd sweat if it gets caught by tests.